### DISCUSSION TITLE
Understanding Pressure Profile in Pyrosim

### DISCUSSION THREAD

**Discussion by Saad-PCEC** [firemodels/fds#13739] [Q&A]
Hi, 
I have just started using Pyrosim. I have successfully designed and simulated the model. However, when I looked at the pressure graph it starts from 0 Pa whereas in reality it should start with atmospheric pressure. In simulation Parameter (Environment Tab) I did tick (activate) the ambient temperature and pressure box but the pressure box gets untick automatically. 

I was wondering if 0Pa represent atmospheric pressure or there is something wrong in my model. Furthermore, I cannot understand why does atmospheric pressure box gets untick automatically ?

**Comment by drjfloyd** [comment]
This forum is for FDS, the softwrare program that Pyrosim makes input files for. Questions on the Pyrosim interface should be directed to Thunderhead Engineering. For FDS questions your first resource should be the FDS User's Guide which will define what all the outputs are.
